front/src/modules/trainschedule/components/Timetable/TrainScheduleItem.tsx

```diff
@@ -267,6 +267,7 @@ const TrainScheduleItem = ({
         duplicateTimetableItem={duplicateTrain}
         editTimetableItem={() => selectTrainToEdit(train)}
         deleteTimetableItem={deleteTrain}
+        isTrainValid={!train.invalidReason}
```

Comment:
```suggestion
        isTimetableItemValid={!train.invalidReason}
```

For homogeneity reasons, should we use the TimetableItem wording?

Reply:
Good catch !
Changed it